src/utils/createAlpineREPL.js

```diff
@@ -0,0 +1,34 @@
+import { getParameters } from 'codesandbox/lib/api/define';
```

Comment:
Wow I didn't know you can do that with codesandbox.
It looks dope ! 🔥

Reply:
You'll have to let me know if we need to make changes here! I had to add `codesandbox` as a dependency. I installed it as a dev dependency assuming it pre-bundles everything and no JavaScript is actually run once it is deployed to the website, but I don't know if this is the case.